packages/platform-ios/src/commands/runIOS/__tests__/findMatchingSimulator-test.js

```diff
@@ -671,7 +671,7 @@ describe('findMatchingSimulator', () => {
             ],
           },
         },
-        'iPhone 6s (10.0)',
```

Comment:
Instead of changing this test, can you please add a new case? I'd like to be sure we don't introduce any regression

Reply:
@thymikee got it, I really didn't think about the support for old Xcode simulator name.
But I agree with you, I think we should make it more compatible with previous version Xcode simulator name because there are quite a lot of developers who still has the old simulator setup.
I will close this pull request first and create another one to make it more compatible with old versions, thanks for your time.